apps/staking/app/mystakes/modules/TotalRewardsModule.tsx

```diff
@@ -27,8 +32,10 @@ export default function TotalRewardsModule(params?: AddressModuleProps) {
 
   const { claimed, status, refetch } = useGetRecipients({ address: address! });
```

Comment:
There are quite a few `address: address!` references in the codebase that biome is complaining about. Would you mind adding `//biome-ignore lint/style/noNonNullAssertion: limitation of the library` to them please? 🙏

Reply:
Biome doesn't flag this issue, it's already an excluded rule. When I (and the CI) runs the command there's no issues flagged.
```
"style": {
  // TODO: resolve this once we have a better solution for wagmi query enabling with address
  "noNonNullAssertion": "off"
}
```